app/src/main/java/com/tvtracker/dto/ImdbResponse.kt

```diff
@@ -2,6 +2,8 @@ package com.tvtracker.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class ImdbResponse(@SerializedName("Search")       var results: ArrayList<MediaItem> = ArrayList<MediaItem>(),
-                        @SerializedName("totalResults") var totalResults: Int = 0,
-                        @SerializedName("Response")     var success: Boolean = false)
+data class ImdbResponse(
+        @SerializedName("Search")       var results: ArrayList<MediaItem> = ArrayList<MediaItem>(),
```

Comment:
Restructured dto signatures to proper coding conventions for readability.
https://developer.android.com/kotlin/style-guide#formatting